src/components/FBanner/FBanner.vue

```diff
@@ -124,27 +127,35 @@ export default {
       };
     }
   },
+  watch: {
+    currentNumber() {
+      if (Math.abs(this.lastCurrentNumber - this.currentNumber) >= 2) {
+        // this.animation = true;
+        // this.currentNumber++;
+      }
+      // this.lastCurrentNumber = this.currentNumber;
+    }
+  },
+  created() {
+    this.classImages = Array.from({ length: this.images.length }, () => null);
```

Comment:
Compreendi. Não precisa alterar, pois o resultado é o mesmo, só expondo que eu utilizo outra forma de fazer esse tipo de preenchimento, que é com `Array.fill()`, por exemplo: `Array(this.images.length).fill(null)`